SRC/GAME.CPP

```diff
@@ -2342,6 +2342,9 @@ void game()
                  for ( a = 0; a < ACTIVE_PLAYERS; a ++  ) aplayer[a]->shop();
             }
 
+            scr_y_size = 240;
+            tk_port_resolution_changed();
```

Comment:
What if `tk_port_resolution_changed` was called `tk_port_change_resolution()` and accepted the new `scr_y_size` as an argument?

Reply:
> 
> What if `tk_port_resolution_changed` was called `tk_port_change_resolution()` and accepted the new `scr_y_size` as an argument (and set it internally)?

Works for me! Done.